src/AppBundle/Validator/TwoPerformancePerDayValidator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AppBundle\Validator;
+
+use Symfony\Component\Validator\ConstraintValidator;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Validator\Constraint;
+
+class TwoPerformancePerDayValidator extends ConstraintValidator
+{
+    private $em;
+
+    public function __construct(EntityManager $em)
+    {
+        $this->em = $em;
+    }
+
+    public function validate($object, Constraint $constraint)
+    {
+        $date = $object->getPremiere()->format('Y-m-d');
```

Comment:
You need work with object (because its about OOP). Pass DateTime object to the repository method